x-pack/legacy/plugins/beats_management/server/rest_api/__tests__/beats_assignments.test.ts

```diff
@@ -1,249 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { CMServerLibs } from '../../lib/types';
-import { HapiBackendFrameworkAdapter } from './../../lib/adapters/framework/hapi_framework_adapter';
-import { testHarnes } from './test_harnes';
-
-describe('assign_tags_to_beats', () => {
-  let serverLibs: CMServerLibs;
-
-  beforeAll(async () => {
-    jest.setTimeout(100000); // 1 second
-
-    serverLibs = await testHarnes.getServerLibs();
-  });
```

Comment:
So, this test suite (the only one of the server-side plugin) was actually working, however:
- It relies on direct access to the HAPI server, both for `injectRequest` and for the custom testing lib composer, which can't be easily converted to KP.
- It's based on fully mocked (custom in memory implementation of the adapter) API, so it's not really testing anything concrete. 
- It only got 4 tests, which are all already covered by the FTR test suite in `x-pack/test/api_integration/apis/beats`